src/content/tutorials/drizzle-with-db/drizzle-with-neon.mdx

```diff
@@ -0,0 +1,241 @@
+---
+title: "Drizzle with Neon Postgres"
+date: "2024-05-10"
+svgs: ['<svg width="160" height="160" viewBox="0 0 160 160" fill="none" xmlns="http://www.w3.org/2000/svg"><rect width="9.63139" height="40.8516" rx="4.8157" transform="matrix(0.873028 0.48767 -0.497212 0.867629 43.4805 67.3037)" fill="currentColor"></rect><rect width="9.63139" height="40.8516" rx="4.8157" transform="matrix(0.873028 0.48767 -0.497212 0.867629 76.9395 46.5342)" fill="currentColor"></rect><rect width="9.63139" height="40.8516" rx="4.8157" transform="matrix(0.873028 0.48767 -0.497212 0.867629 128.424 46.5352)" fill="currentColor"></rect><rect width="9.63139" height="40.8516" rx="4.8157" transform="matrix(0.873028 0.48767 -0.497212 0.867629 94.957 67.3037)" fill="currentColor"></rect></svg>', <svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M13.3788 21.6677C12.8679 22.3111 11.8319 21.9586 11.8196 21.137L11.6396 9.12061H19.7194C21.1828 9.12061 21.9991 10.8109 21.089 11.9571L13.3788 21.6677Z" fill="url(#paint0_linear_142_2)"/><path d="M13.3788 21.6677C12.8679 22.3111 11.8319 21.9586 11.8196 21.137L11.6396 9.12061H19.7194C21.1828 9.12061 21.9991 10.8109 21.089 11.9571L13.3788 21.6677Z" fill="url(#paint1_linear_142_2)" fill-opacity="0.2"/><path d="M10.0928 2.33232C10.6037 1.68882 11.6397 2.04141 11.652 2.86299L11.7309 14.8794H3.75221C2.2887 14.8794 1.47247 13.1891 2.38252 12.0429L10.0928 2.33232Z" fill="currentColor"/><defs><linearGradient id="paint0_linear_142_2" x1="11.6396" y1="11.7849" x2="18.8206" y2="14.7967" gradientUnits="userSpaceOnUse"><stop stop-color="currentColor"/><stop offset="1" stop-color="currentColor"/></linearGradient><linearGradient id="paint1_linear_142_2" x1="8.45585" y1="7.42589" x2="11.7308" y2="13.5908" gradientUnits="userSpaceOnUse"><stop/><stop offset="1" stop-opacity="0"/></linearGradient></defs></svg>]
+---
+
+import Prerequisites from "@mdx/Prerequisites.astro";
+import Npm from "@mdx/Npm.astro";
+import Steps from "@mdx/Steps.astro";
+import Section from "@mdx/Section.astro";
+import Callout from "@mdx/Callout.astro";
+
+This tutorial demonstrates how to use Drizzle ORM with [Neon Postgres](https://neon.tech/) database. 
+
+<Prerequisites>
+  - `Neon`, a serverless postgres database. If you do not have an existing Neon account, sign up [here](https://neon.tech). 
+    
+    Your Neon projects come with a ready-to-use Postgres database named `neondb`. We'll use this database in the following examples.
+  
+  - You should have installed Drizzle ORM and [Drizzle kit](https://orm.drizzle.team/kit-docs/overview) along with the [Neon serverless driver](https://neon.tech/docs/serverless/serverless-driver). You can do this by running the following command:
+  <Npm>
+    drizzle-orm 
+    -D drizzle-kit
+    @neondatabase/serverless
+  </Npm>
+  
+  - You should have installed the `dotenv` package for managing environment variables, and `tsx` to run typescript files. 
+  <Npm>
+    dotenv
+    tsx
```

Comment:
If it is used only for applying migrations then you can remove it as we use `drizzle-kit migrate` command